### ISSUE TITLE
Controls/KB&M not working

### ISSUE THREAD

**Issue by mapyl** [xoxor4d/p2-rtx#23]
Not sure exactly what would cause this, but my controls don't work in game, nor the main menu. I can open the remix menu w/ alt-x and change options on there. 

There is rare moments where my controls temporarily work, but just randomly cut out in game after ~3-4 minutes, resulting in my mouse getting stuck in the top-right corner of my monitor when attempting to quit the game.

**Comment by xoxor4d** [comment]
And that does not happen when playing the stock version of Portal 2?

**Comment by mapyl** [comment]
It does not, only w/ this here. (I went through a few clean installs as well just to make sure)

**Comment by xoxor4d** [comment]
Same issue on portal 1 rtx? Have you tried other remix mods?
Do you maybe have a controller connected that might cause that?

**Comment by mapyl** [comment]
Portal 1 RTX works fine, controls work on it.
No other peripheral devices are connected except KB&M, I tried with different keyboards as well.

I don't have any virtual controller drivers installed as well.

**Comment by xoxor4d** [comment]
Well they've made some changes regarding IO a while back which might not yet be in P1 RTX. 
Are you in fullscreen mode?

You could try playing with a few input related settings in the bridge.conf (found in `bin/.trex`) .. but make sure to grab the most recent file first: https://github.com/NVIDIAGameWorks/bridge-remix/blob/c40116b77591de70133862c34e1a68782f8e4567/bridge.conf

eg:
- `client.hookMessagePump`
- `client.DirectInput.forward.mousePolicy`


If you somehow get it working, make sure to turn on `exposeRemixApi`
by removing the `#` infront and setting the value to `True` (`exposeRemixApi = True`)

**Comment by mapyl** [comment]
Yeah, no luck... Been messing with it on my free-time and haven't been able to get it to work

**Comment by SmoothWolf20** [comment]
So nobody figured this out??
